pkg/components/row/row.go

```diff
@@ -24,22 +24,26 @@ func New(height float64) core.Row {
 	}
 }
 
+// SetConfig sets the row configuration.
 func (r *row) SetConfig(config *entity.Config) {
 	r.config = config
 	for _, cols := range r.cols {
 		cols.SetConfig(config)
 	}
 }
 
+// Add is responsible to add a core.Col to a core.Row.
```

Comment:
// Add is responsible to add one or more core.Col to a core.Row.

Reply:
done